lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -500,7 +502,13 @@ class GloballyPositionedTextEditingStrategy extends DefaultTextEditingStrategy {
   @override
   void placeElement() {
     super.placeElement();
-    _geometry?.applyToDomElement(domElement);
+    if (hasAutofillGroup) {
+       _geometry?.applyToDomElement(focusedFormElement());
+       placeForm();
+       focusedFormElement().focus();
```

Comment:
Could you please add a comment explaining why we need to call focus here?

Reply:
done

Reply:
Please have a look, I can add more details if it is not clear.